capuccino/capuccino_py.py

```diff
@@ -0,0 +1,28 @@
+from math import *
```

Comment:
Wildcard imports (from <module> import *) should be avoided
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
amended